math/netgen/Portfile

```diff
@@ -91,7 +92,7 @@ foreach pdv ${pythonversions} {
     }
 }
 if {!${any_python}} {
-    default_variants    +python39
+    default_variants    +python311
```

Comment:
this should match the python PG which is still 3.10

Reply:
Thank you for catching this.

Reply:
on the other hand, if this PR stays in the queue for another two days you're okay with `python311` as the change in the `python` PG will happen on January 1 ;)